src/components/MenuItem.js

```diff
@@ -41,43 +41,63 @@ const defaultProps = {
     interactive: true,
 };
 
-const MenuItem = props => (
-    <Pressable
-        onPress={(e) => {
```

Comment:
This change is not acceptable. We don't have to touch the MenuItem for this. Instead, use the ref to calculate whatever is needed on the parent components that use it.

Reply:
Hi @srdjanrist, could you please address this comment so we can move forward :)